scripts/docker-compose/imports/openldap/zac-scripts/zac-ldap-setup-template.ldif

```diff
@@ -73,7 +73,7 @@ objectClass: organizationalPerson
 objectClass: person
 objectClass: top
 cn: testuser1
-sn: User1
+sn: User1 Špëçîâl Characters
```

Comment:
This one is perhaps not needed and coming from a previous PR?

Reply:
Ah yes, I started with this from that other PR branch. I will merge this one to that branch then.

Reply:
fixed now